server/service.go

```diff
@@ -304,6 +310,14 @@ func (m *BoostService) handleGetHeader(w http.ResponseWriter, req *http.Request)
 	m.bids[bidKey(slot, result.bidInfo.blockHash)] = result
 	m.bidsLock.Unlock()
 
+	// Decide response content type (JSON by default)
+	clientAccepts := goacceptheaders.Parse(req.Header.Get("Accept"))
```

Comment:
should be moved up to where proposer accept headers are parsed, at line 272

Reply:
Also, should use the `proposerAcceptContentTypes` variable instead of parsing again

Reply:
> should be moved up to where proposer accept headers are parsed, at line 272

This feels a little weird. I dislike that we have two very similar variables. I will think about this for a minute.